overrides.go

```diff
@@ -67,21 +72,24 @@ func (h Handler) OverridesLookup(asn string) (string, error) {
 	return override.Name, nil
 }
 
-// OverridesSet stores a user defined description for a given ASN
+// OverridesSet stores or updates a user defined description for a given ASN
 // in the database of local overrides.
 func (h Handler) OverridesSet(asn string, descr string) error {
 	if h.overrides == nil {
 		return OverridesNilCollectionError
 	}
+	if !reASN.MatchString(asn) {
+		return OverridesMalformedAsnError
+	}
 	_, err := h.overrides.UpsertId(asn, bson.M{"$set": bson.M{"name": descr}})
 	if err != nil {
 		return fmt.Errorf("cannot set override: %s", err)
 	}
 	return nil
 }
 
-// OverridesRemove removes the description for a given ASN
-// from the database of local overrides.
+// OverridesRemove makes sure the description for a given ASN
+// is removed from the database of local overrides.
 func (h Handler) OverridesRemove(asn string) error {
```

Comment:
Sorry bad choice of words I guess

I wanted to say thar OverridesRemove() may be called with an unexistent asn. It won't return an error in this case.